config/functions/bootstrap.js

```diff
@@ -0,0 +1,48 @@
+module.exports = async () => {
+  // Check if the plugin users-permissions is installed because the comments needs it
+  if (Object.keys(strapi.plugins).indexOf('users-permissions') === -1) {
+    throw new Error(
+      'In order to make the comments plugin works the users-permissions one is required'
+    );
+  }
+
+  // Add permissions
+  const actions = [
+    {
+      section: 'plugins',
+      displayName: 'Access the Comments',
+      uid: 'read',
+      pluginName: 'comments',
+    },
+    {
+      section: 'plugins',
+      displayName: 'Modefation',
+      uid: 'moderate',
+      pluginName: 'comments',
+    },
+    {
+      section: 'plugins',
+      displayName: 'Single comments',
+      uid: 'moderate.block.comment',
```

Comment:
these UUIDs are repeated here, in `pluginPermissions` and in `routes.json`
maybe extract them to constants?

Reply:
good point, will extract them

Reply:
`config/functions/bootstrap.js` and `admin/src/permissions.js` are in differen projects (NodeJS and React), there is no possibility for them to import from each other because of how strapi is built. So even then, there will be duplication.

Current way is the common one (I know that not so handy).